src/Audit/CacheBinBackend.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drutiny\Plugin\Drupal8\Audit;
+
+use Drutiny\Audit\AbstractAnalysis;
+use Drutiny\Sandbox\Sandbox;
+use Drutiny\Annotation\Param;
+
+/**
+ * @Param(
+ *  name = "expression",
+ *  type = "string",
+ *  default = "true",
+ *  description = "The expression language to evaludate. See https://symfony.com/doc/current/components/expression_language/syntax.html"
+ * )
+ * @Param(
+ *  name = "bin_backends",
+ *  description = "Key value pair of cache bins and expected bin backend where key is bin and value is backend",
+ *  type = "array"
+ * )
+ */
+class CacheBinBackend extends AbstractAnalysis {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function gather(Sandbox $sandbox) {
+    // Get site's cache bin backend and all settings.
+    list($cache_bin_backends, $site_settings) = $sandbox->drush()->evaluate(function () {
+      return [
+        \Drupal::getContainer()->getParameter('cache_default_bin_backends'),
```

Comment:
tried to get all container params instead one but container only supports to get param by name and not all.

Reply:
See https://github.com/symfony/symfony/blob/3.3/src/Symfony/Component/DependencyInjection/Container.php#L123

I think you can use ` \Drupal::getContainer()->getParameterBag()->all()`

Reply:
`Container:: getParameterBag ()` is symfony implementation and not part of the contract (ContainerInterface) and thus not available in drupal.